.buildkite/e2e/reporter/templates/notify-failures.tpl.yml

```diff
@@ -11,8 +11,8 @@ steps:
             - "#eck"
           message: |
           {{- range $envName, $sortedTests := .Tests }}
-            {{- range $test := $sortedTests.Failed }}
-            🐞 `{{ splitTestName $test.Name }}` {{ $envName }}
+            {{- range $test := $sortedTests.ShortFailed }}
```

Comment:
Should this not be in  the `annotate-failures.tpl.md` instead of this one? Because the as I understand it the Slack notification actually worked and does not seem to suffer from this limitation.